modules/sbo-configuration-of-directory-path-to-project-binding-data.adoc

```diff
@@ -0,0 +1,68 @@
+[id="sbo-configuration-of-directory-path-to-project-binding-data_{context}"]
+= Configuration of the directory path to project the binding data inside workload container
+
+For projecting the binding data, by default, files are mounted at a specific directory in your workload resource. You can configure the directory path using the existing `SERVICE_BINDING_ROOT` environment variable setup in the container where your workload runs.
+
+.Example
+----
+$SERVICE_BINDING_ROOT <1>
+├── account-database <2>
+│   ├── type <3>
+│   ├── provider <4>
+│   ├── uri
+│   ├── username
+│   └── password
+└── transaction-event-stream <2>
+    ├── type
+    ├── connection-count
+    ├── uri
+    ├── certificates
+    └── private-key
+----
+<1> Root directory.
+<2> Directory that stores the binding data.
+<3> Mandatory identifier that identifies the type of the binding data projected into the corresponding directory.
+<4> An optional identifier to identify the provider so that the application can identify the type of  backing service it can connect to.
+
+For consuming binding data as environment variables, you can use the built-in language feature of your programming language of choice to read environment variables.
+
+.Example: Python client usage
+----
+import os
+username = os.getenv("USERNAME")
+password = os.getenv("PASSWORD")
+----
+
+== Computation of the final path for projecting the binding data as files
+
+The following table summarizes the configuration of how the final path for the binding data projection is computed when files are mounted at a specific directory, by default:
```

Comment:
```suggestion
The following table summarizes the configuration of how the final path for the binding data projection is computed when files are mounted at a specific directory:
```
IMHO, not necessary to mention here that the the binding data is mounted as files by default. The table sais what happens in a situation when SBO mounts as files. For the context of this table, It doesn't matter if that is the default behaviour of SBO or not.